### PR TITLE
Removed all hardcoded strings

### PR DESCRIPTION
**Pull request by smithhug**
All hardcoded strings have been added to strings.xml. Searching the project for hardcoded text will reveal 0 results now. There are no duplicate strings.

### CODE REVIEW

File: app/src/main/res/layout/activity_care_plan.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
                 android:layout_width="750dp"
                 android:layout_height="61dp"
                 android:layout_marginTop="45dp"
-                android:contentDescription="TODO"
+                android:contentDescription="@string/todo"
```
Comment (el-veee) [inline app/src/main/res/layout/activity_care_plan.xml]: I would leave out the TODO ones. Adding appropriate content descriptions to our views will be a separate piece of work, and putting this string here might make android studio think this has a proper content description, when it doesn't

File: app/src/main/res/layout/activity_care_plan_summary.xml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:layout_width="fill_parent"
+        xmlns:tools="http://schemas.android.com/tools"
```
Comment (el-veee) [inline app/src/main/res/layout/activity_care_plan_summary.xml]: Weird formatting?

File: app/src/main/res/layout/activity_caretaker_dashboard.xml [hunk 3]
```diff
@@ -41,7 +41,7 @@
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
                     android:layout_marginTop="54dp"
-                    android:contentDescription="TODO"
+                    android:contentDescription="@string/todo"
```
Comment (el-veee) [inline app/src/main/res/layout/activity_caretaker_dashboard.xml]: Same thing about the TODO string

**Review by el-veee — CHANGES_REQUESTED** [review]
Couple minor things, but otherwise looks good

**Comment by smithhug** [comment]
Removed todo string and replaced referenced with "TODO"

File: app/src/main/res/layout/activity_care_plan_summary.xml (hunk 2 above)
Comment (smithhug) [reply app/src/main/res/layout/activity_care_plan_summary.xml]: Fixed formatting in recent commit.

**Review by el-veee — APPROVED** [review]

**Pull request merged**